blog.html

```diff
@@ -22,6 +22,119 @@
         <!--#include virtual="includes/_nav.htm" -->
         <div class="right">
             <h1 class="content-title">Blog</h1>
+            <article>
+                <h2 class="bullet">
+                    <a id="apache_kafka_370_release_announcement"></a>
+                    <a href="#apache_kafka_370_release_announcement">Apache Kafka 3.7.0 Release Announcement</a>
+                </h2>
+                TODO: January 2024 - Stanislav Kozlovski (<a href="https://twitter.com/0xeed">@BdKozlovski</a>)
```

Comment:
I assume we have to edit & merge this on the day of the announcement, hence left a "TODO:" here

Reply:
You also want to fix the twitter link to point to your profile

Reply:
thanks